src/actions/productsActions.js

```diff
@@ -0,0 +1,24 @@
+export const getProducts = (products) =>{
```

Comment:
Need to have the following states:
```
Loading
Success
Failure
```
will need to dispatch actions and name them accordingly.
what is expected to happen is that a loader will show till the records are fetched and then on records fetched the loader will dissapear...
incase of any issue with the Products api, the UI should handle the failure gracefully, my showing a message like `Oops we are currently facing issues, please try again later`